src/trix/models/attachment.coffee

```diff
@@ -145,9 +150,16 @@ class Trix.Attachment extends Trix.Object
   preload: (url, callback) ->
     if url and url isnt @preloadedURL
       @preloadedURL ?= url
-      operation = new Trix.ImagePreloadOperation url
-      operation.then ({width, height}) =>
-        @preloadedURL = url
-        @setAttributes({width, height})
-        @previewDelegate?.attachmentDidPreload?()
-        callback?()
+      if @getContentType() == 'text/html'
```

Comment:
these `text/html` checks are all pretty rough. if this is something that would actually be useful I can refactor these.